examples/selenium-container/src/test/java/SeleniumContainerTest.java

```diff
@@ -45,7 +45,7 @@ public void simplePlainSeleniumTest() {
         driver.get("http://host.testcontainers.internal:" + port + "/foo.html");
         List<WebElement> hElement = driver.findElementsByTagName("h");
 
-        assertTrue("The h element is found", hElement != null && hElement.size() > 0);
+        assertThat(hElement != null && hElement.size() > 0).as("The h element is found").isTrue();
```

Comment:
nit: I know that it was automated, but could probably use AssertJ's support for lists